Mathlib/Algebra/Homology/HomotopyCategory/HomComplex.lean

```diff
@@ -572,6 +590,12 @@ lemma ext (z₁ z₂ : Cocycle F G n) (h : (z₁ : Cochain F G n) = z₂) : z₁
 lemma ext_iff (z₁ z₂ : Cocycle F G n) : z₁ = z₂ ↔ (z₁ : Cochain F G n) = z₂ :=
   Subtype.ext_iff
 
+instance : SMul R (Cocycle F G n) where
+  smul r z := ⟨r • z.1, by
+    have hz := z.2
+    rw [mem_iff n (n + 1) rfl] at hz ⊢
+    simp only [δ_smul, hz, smul_zero]⟩
```

Comment:
This probably creates an instance diamond when `R = Int`, though I suspect fixing it is a problem for another time.